crates/primitives/src/bloom.rs

```diff
@@ -69,4 +120,19 @@ mod tests {
             ))
         );
     }
+    #[test]
+    fn arbitrary() {
+        proptest::proptest!(|(bloom: Bloom)| {
+            let mut buf = vec![];
+            bloom.to_compact(&mut buf);
+
+            // Add noise
+            buf.push(1);
```

Comment:
should the noise also be randomly provided by proptest?

Reply:
i'd say it's not necessary in this case. the goal is to make sure that only 256 bytes are consumed, and not more.